mdz/pkg/server/openmodelz_install.go

```diff
@@ -113,5 +129,12 @@ func (s *openModelZInstallStep) Verify() error {
 	}
 
 	resultDomain = re.FindString(string(output))
+
+	// If enabled modelzcloud control plane, you need make configuration
+	if s.options.ModelZCloud.Enabled {
+		if s.options.ModelZCloud.URL == "" || s.options.ModelZCloud.Token == "" || s.options.ModelZCloud.Region == "" {
+			return fmt.Errorf("modelzcloud configuration is not complete")
+		}
+	}
```

Comment:
Shall we verify this before we apply the helm chart?

Reply:
sgtm